client/src/components/add-job-post-component/index.jsx

```diff
@@ -0,0 +1,282 @@
+import { useState, useRef } from "react";
+import { useNavigate } from "react-router-dom";
+import PropTypes from "prop-types";
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogContentText,
+  DialogTitle,
+  Box,
+  Stack,
+  Typography,
+  Pagination,
+} from "@mui/material";
+import AddJobDetails from "./job-info-form";
+import AddApplicationFields from "./application-form-fields";
+import { Container, ButtonContainer } from "./index.styles";
+import UserType from "../../prop-types/UserType";
+import createJobPost from "./mockResponse";
+import PostingResultDialog from "./posting-result-dialog";
+
+function AddJobLead({ jobPostData, updateJobPostData, currUser }) {
```

Comment:
I see you've used both JobLead and JobPost while naming your components, just wanted to check what you mean by JobLead.